sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 6.0.0-beta.3 (Unreleased)
+## 6.0.0-beta.3 (2022-11-04)
```

Comment:
How about adding the information about spring-boot version and spring-cloud version.

Reply:
This info has been added here https://github.com/moarychan/azure-sdk-for-java/blob/release/spring-cloud-azure-6.0.0-beta.3/sdk/spring/CHANGELOG.md#600-beta3-2022-11-04